### PR TITLE
Buy-Skycoin-Over-The-Counter.de.md

### PR DESCRIPTION
**Pull request by kietob**
German translation

### CODE REVIEW

File: content/infographics/Buy-Skycoin-Over-The-Counter.de.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
++++
+title = "Kaufe Skycoin direkt (Over the counter) [INFOGRAPHIC]"
```
Comment (IamVanCleef) [inline content/infographics/Buy-Skycoin-Over-The-Counter.de.md]: Skycoins direkt kaufen

File: content/infographics/Buy-Skycoin-Over-The-Counter.de.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+tags = [
+    "OTC",
+    "Buy Skycoin",
+    "Infographics",
+    "Infografik",
+    "Skycoin kaufen",
```
Comment (IamVanCleef) [inline content/infographics/Buy-Skycoin-Over-The-Counter.de.md]: Skycoins

File: content/infographics/Buy-Skycoin-Over-The-Counter.de.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Es ist jetzt möglich, Skycoin direkt von skycoin.net zu kaufen.
+
+* Besuche https://exchange.skycoin.net
+* Wähle eine der verfügbaren Währungen
+* Gebe die Anzahl ein & klicke auf weiter
+* Gebe die Adresse ein
+* Klicke auf Exchange
+
+> Wenn du Skycoins direkt über unsere Webseite kauft, werden die Coins automatisch über den Exchange zum günstigsten Preis gekauft. Die Transaktionen werden von Coinswitch durchgeführt. Unsere Partner umgehen den Ärger mit den Exchanges. Nie wieder Auszahlungsgebühren!
```
Comment (C3POv7) [inline content/infographics/Buy-Skycoin-Over-The-Counter.de.md]: "umgehen damit die Probleme" anstelle von "den Ärger"

File: content/infographics/Buy-Skycoin-Over-The-Counter.de.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    url = "img/Buy-Skycoin-Over-The-Counter.jpg"
+    type = "image/jpg"
+    length = 302249
++++
+
+## Es ist jetzt möglich, Skycoin direkt von skycoin.net zu kaufen.
+
+* Besuche https://exchange.skycoin.net
+* Wähle eine der verfügbaren Währungen
+* Gebe die Anzahl ein & klicke auf weiter
```
Comment (C3POv7) [inline content/infographics/Buy-Skycoin-Over-The-Counter.de.md]: die Anzahl an Coins

**Comment by kietob** [comment]
i really do not understand why the checks fails suddenly after changing two words...

**Comment by gz-c** [comment]
the failure is unrelated to your changes, im fixing it

**Comment by kietob** [comment]
test commit if problems are solved

**Comment by kietob** [comment]
do I have to do anything or will the red cross disappear by itself at some point?

**Comment by gz-c** [comment]
If you merge/rebase with the master branch they'll go away, but if you don't know how to do that you can ignore it and it'll be fine

**Comment by kietob** [comment]
i am sorry, I do not know how to rebase. I read about it, but still it wasn't clear to me. Thanks!

**Pull request closed without merging**